===================================================================
RCS file: /usr/local/src/cvsroot/nefu/configure.ac,v
retrieving revision 1.14
retrieving revision 1.15
diff -u -r1.14 -r1.15
--- configure.ac	2002/12/09 23:43:15	1.14
+++ configure.ac	2003/11/20 18:34:51	1.15
@@ -5,8 +5,8 @@
 AC_COPYRIGHT([Copyright (c) 1998 Regents of The University of Michigan])
 AC_CONFIG_SRCDIR([libtest/tcp.c])
 AC_CONFIG_AUX_DIR([libsnet])
-#AC_REVISION($Revision: 1.1 $)
-#AC_CONFIG_HEADER([config.h])
+#AC_REVISION($Revision: 1.1 $)
+AC_CONFIG_HEADER([config.h])
 
 # Local Stuff
 AC_PREFIX_DEFAULT(/usr/local/nefu)
@@ -10,7 +10,17 @@
 
 # Local Stuff
 AC_PREFIX_DEFAULT(/usr/local/nefu)
-AC_SUBST(html_dir, [\${prefix}/public_html])
+AC_ARG_WITH(htmldir,
+    [  --with-htmldir[=PATH] directory where HTML will be written],
+    html_dir=$with_htmldir,
+    html_dir=[\${prefix}/public_html])
+AC_ARG_WITH(scriptdir,
+    [  --with-scriptdir[=PATH] directory where shell tests are stored],
+    script_dir=$with_scriptdir,
+    script_dir=[\${prefix}/shelltests])
+
+AC_SUBST(html_dir, [${html_dir}])
+AC_SUBST(script_dir, [${script_dir}])
 AC_ARG_ENABLE(html,
 	[AC_HELP_STRING([--disable-html],
 		[do not output html or install html files])],
